<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigStationParametre</class>
 <widget class="QDialog" name="ConfigStationParametre">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration Nouveaux Elements</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" rowspan="2" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nouveaux Paramètres/Stations</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QListView" name="listViewNouveaux"/>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="0">
      <widget class="QComboBox" name="comboBoxCategorie"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Catégorie</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="lineEditNom">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Ajouter Categorie</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLineEdit" name="lineEditNewCategorie"/>
     </item>
     <item row="11" column="0">
      <widget class="QPushButton" name="buttonValiderCategorie">
       <property name="text">
        <string>Valider la catégorie</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="buttonFin">
     <property name="text">
      <string>Fin</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
